application/src/main/java/org/togetherjava/tjbot/commands/modmail/ModmailCommand.java

```diff
@@ -0,0 +1,138 @@
+package org.togetherjava.tjbot.commands.modmail;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.SelectionMenuEvent;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.components.ActionRow;
+import net.dv8tion.jda.api.interactions.components.selections.SelectOption;
+import net.dv8tion.jda.api.interactions.components.selections.SelectionMenu;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+import org.togetherjava.tjbot.config.Config;
+
+import java.util.*;
+
+public class ModmailCommand extends SlashCommandAdapter {
+    private static final Logger logger = LoggerFactory.getLogger(ModmailCommand.class);
+
+    private static final String COMMAND_NAME = "modmail";
+    private static final String TARGET_OPTION = "message";
+    private static final Config config = Config.getInstance();
+
+    static final List<SelectOption> mods = new ArrayList<>();
+    static final Map<String, User> modsMap = new HashMap<>();
```

Comment:
It shouldn't be done this way, keep the variables private, use the onReady event to load the options/mods.

So keep it within this class, make the variables private and don't make them static.